cpp/tests/strings/integers_tests.cu

```diff
@@ -137,7 +138,7 @@ TEST_F(StringsConvertTest, HexToInteger)
     for( auto itr = h_strings.begin(); itr != h_strings.end(); ++itr )
     {
         if( *itr==nullptr )
-            h_expected.push_back(0);
+           h_expected.push_back(0);
         else
            h_expected.push_back( (int)std::stol(std::string(*itr),0,16) );
```

Comment:
```suggestion
            h_expected.push_back( (int)std::stol(std::string(*itr),0,16) );
```
Shouldn't this line have been indented by 1 instead?

Reply:
You are right.